README.md

```diff
@@ -277,6 +277,10 @@ Licensed under the [GNU General Public License, Version 3](LICENSE).
         * Adapt your environment using the control panel (just search `setting environment windows control panel` using your favorite search engine)
         * Adapt your start (.bat) file launching ethminer and add a line `set CUDA_DEVICE_ORDER=PCI_BUS_ID` or `setx CUDA_DEVICE_ORDER PCI_BUS_ID`. For more info about `set` see [here](https://docs.microsoft.com/en-us/windows-server/administration/windows-commands/set_1), for more info about `setx` see [here](https://docs.microsoft.com/en-us/windows-server/administration/windows-commands/setx)
 
+10. Error: Insufficient CUDA driver: 9010
+
+    On Linux, you have to upgrade your Nvidia driver. Install `nvidia-396` package or newer.
```

Comment:
I wouldn't limit this on Linux because it's unrelated to the OS.

Reply:
Not sure we should force this upgrade to nvidia-396... The driver has issues with the 'nvidia-settings' command used for overclocking. Any use of the command will knock out all but the 1st GPU in a multi-gpu rig. Hope this is just a problem on my systems, or the overclockers will not be happy with this!

Reply:
Apparently only a problem on my systems...

Reply:
> Any use of the command will knock out all but the 1st GPU in a multi-gpu rig

On linux never experienced that and I'm on 396.

Reply:
I agree that upgrading to CUDA 9.2 was a mistake. But I prefer to leave it as it is because the Nvidia drivers situation will improve over time.